include/dpp/appcommand.h

```diff
@@ -601,26 +601,12 @@ struct DPP_EXPORT command_interaction {
  */
 void from_json(const nlohmann::json& j, command_interaction& ci);
 
-/**
- * @brief Component type, either button or select
- */
-enum component_type_t {
-	/**
-	 * @brief Button
-	 */
-	cotype_button = 2,
-	/**
-	 * @brief Option select list (drop-down)
-	 */
-	cotype_select = 3
-};
-
```

Comment:
`dpp::component_type` is now used instead because it was just a duplicate of it.